sentry-android-core/build.gradle.kts

```diff
@@ -24,10 +24,9 @@ android {
     }
 
     buildTypes {
-        getByName("debug") {
-            isMinifyEnabled = false
-        }
+        getByName("debug")
         getByName("release") {
+//            isMinifyEnabled = true // Do we want to enable it for our lib?
```

Comment:
@bruno-garcia can you give input about that?

Reply:
IMHO it's best we don't minify be default. We're not trying to obfuscate anything, code is OSS and if they proguard their app it will do so fo the lib.